packages/utils/src/validators.ts

```diff
@@ -0,0 +1,13 @@
+const URL_REGEX =
+    /^(http|ws)s?:\/\/[a-z0-9]([a-z0-9.-]+)?(:[0-9]{1,5})?((\/)?(([a-z0-9-_])+(\/)?)+)$/i;
+
+export const UPPERCASE_RE = new RegExp('^(.*[A-Z].*)$');
+
+export const isUrl = (value: string): boolean => URL_REGEX.test(value);
+
+export function isASCII(value?: string): boolean {
```

Comment:
nit: I think we should rename it to `isAscii`, I did brief research and most libraries use it this way... but no big deal, resolve if you don't want to change it

Reply:
Yes, probably if we follow the same style as the variable above `isUrl` where both are abbreviations it should be named as you suggested. :+1: